Dockerfile

```diff
@@ -2,18 +2,20 @@ FROM oven/bun AS builder
 
 WORKDIR /app
 
+ARG PUB_ENV
 ARG PUB_HOSTNAME
 ARG PUB_PLAUSIBLE_URL
 
+ENV PUB_ENV=${PUB_ENV}
 ENV PUB_HOSTNAME=${PUB_HOSTNAME}
 ENV PUB_PLAUSIBLE_URL=${PUB_PLAUSIBLE_URL}
 
 COPY package.json ./
 
-RUN bun install
-
 COPY . ./
 
+RUN bun install
+
```

Comment:
copying the `patches` folder should be fine right?

the idea of copying `package.json` first, installing dependencies and then copying and building the rest of the code is so that the phase/layer that installs dependencies is cached and doesn't have to be run again each time a line of code changes (which would increase the image build time).

something like this is what i mean:
```dockerfile
COPY package.json ./
COPY patches/ ./
RUN bun install

COPY . ./
RUN bun run build
```

Reply:
sorry, the reason it didn't work was actually my fault: it's because the `COPY patches/ ./` instruction is copying all files from `patches/` to the working directory (`/app`).

it should be like this instead:
```dockerfile
COPY package.json ./
COPY patches/ ./patches

RUN bun install

COPY . ./

RUN bun run build
```
i've tested this on my home server and everything seems alright

Reply:
you're right it's working, i was too focused on the bun install that i didn't notice the folders :D